src/python/essentia/pytools/spectral.py

```diff
@@ -243,3 +244,87 @@ def nsgcq_overlap_add(cq):
         cqOverlap[:, jj * hopSize + index] += cq[jj]
 
     return cqOverlap[:, hopSize:]
+
+
+def hpcp(audio, sampleRate=44100, frameSize=4096, hopSize=2048, windowType='blackmanharris62', 
```

Comment:
Check if we can use ```**kwargs```` and refer to the algorithms documentation to declutter the list of all available parameters.

Reply:
added a ```hpcpgram``` function which computes hpcp for overlapped frames of a given input audio signal.  Used ```**kwargs``` to pass additional parameters of essentia standard mode HPCP algorithm.